src/test/java/spring/boot/com/sbparser/DataParseTest.java

```diff
@@ -31,17 +30,20 @@
         expectedReview.setHelpfulnessNumerator(1);
         expectedReview.setHelpfulnessDenominator(1);
         expectedReview.setScore(5);
-        expectedReview.setTime(LocalDateTime.ofInstant(Instant.ofEpochSecond(1107820800), ZoneId.systemDefault()));
+        expectedReview.setTime(LocalDateTime
+                .ofInstant(Instant.ofEpochSecond(1107820800), ZoneId.systemDefault()));
         expectedReview.setSummary("The Best Hot Sauce in the World");
-        expectedReview.setText("I don't know if it's the cactus or the tequila or just the unique combination "
-                + "of ingredients, but the flavour of this hot sauce makes it one of a kind!  We picked up a bottle "
-                + "once on a trip we were on and brought it back home with us and were totally blown away!  When we "
-                + "realized that we simply couldn't find it anywhere in our city we were bummed.<br /><br />Now, "
-                + "because of the magic of the internet, we have a case of the sauce and are ecstatic because of it."
-                + "<br /><br />If you love hot sauce..I mean really love hot sauce, but don't want a sauce that "
-                + "tastelessly burns your throat, grab a bottle of Tequila Picante Gourmet de Inclan.  Just "
-                + "realize that once you taste it, you will never want to use any other sauce.<br /><br />"
-                + "Thank you for the personal, incredible service!");
+        expectedReview.setText("I don't know if it's the cactus or the tequila or just the unique "
+                + "combination of ingredients, but the flavour of this hot sauce makes it one of a "
+                + "kind!  We picked up a bottle once on a trip we were on and brought it back home "
+                + "with us and were totally blown away!  When we realized that we simply couldn't "
+                + "find it anywhere in our city we were bummed.<br /><br />Now, because of the "
+                + "magic of the internet, we have a case of the sauce and are ecstatic because of "
+                + "it.<br /><br />If you love hot sauce..I mean really love hot sauce, but don't "
+                + "want a sauce that tastelessly burns your throat, grab a bottle of Tequila "
+                + "Picante Gourmet de Inclan.  Just realize that once you taste it, you will never"
+                + " want to use any other sauce.<br /><br />Thank you for the personal,"
+                + " incredible service!");
     }
```

Comment:
let's use `@BeforeAll/BeforeClass` method instead of the static block

Reply:
done